backend/coins/btc/account.go

```diff
@@ -752,7 +752,8 @@ func (account *Account) VerifyAddress(addressID string) (bool, error) {
 	if address == nil {
 		return false, errp.New("unknown address not found")
 	}
-	canVerifyAddress, _, err := account.Keystores().CanVerifyAddresses(account.signingConfiguration, account.Coin())
+	canVerifyAddress, _, err := account.Keystores().CanVerifyAddresses(
+		account.signingConfiguration.Multisig(), account.Coin())
```

Comment:
How about this:

```go
!account.signingConfiguration.Multisig() && account.Keystores().CanVerifyAddresses(account.Coin())
```

Reply:
that would preclude e.g. the bitbox02 from verifying multisig addresses.

Reply:
it just the multisig bool arg is either unused or always negated in various implementations of CanVerifyAddresse. feels a bit wasteful.

Reply:
Found a simpler solution: disable multisig for the bb01. If needed, the bb02 would be a better device for multisig anyway, where this arg would not be needed.

Reply:
:+1: nice